lib/Controller/PageController.php

```diff
@@ -302,6 +303,57 @@ public function index(string $token = '', string $callUser = '', string $passwor
 		return $response;
 	}
 
+	/**
+	 * @PublicPage
+	 * @NoCSRFRequired
+	 * @UseSession
```

Comment:
session is not used?

Reply:
Oops... copy-paste leftover :see_no_evil: